src/wallet/sqlite.cpp

```diff
@@ -23,7 +23,13 @@
 namespace wallet {
 static constexpr int32_t WALLET_SCHEMA_VERSION = 0;
 
-static GlobalMutex g_sqlite_mutex;
+/**
+ * This mutex protects SQLite initialization and shutdown.
+ * sqlite3_config() and sqlite3_shutdown() are not threadsafe (sqlite3_initialize() is).
+ * Concurrent threads that execute SQLiteDatabase::SQLiteDatabase() should have just one
```

Comment:
I'm not sure what you meant with this sentence. `g_sqlite_count` can be higher than one.

Reply:
Changed "one" to "one of them".